tests/unit/visualise/test_dataset_renderer.py

```diff
@@ -215,14 +215,11 @@ def test_diurnal_averaged_data(self, multiday_data):
         diurnal_data = dr.TimeSeries(multiday_data).diurnal_average()
         assert isinstance(diurnal_data, Cube)
 
-# TODO: Refine this test to its simplest form and get it working (i.e. get all track methods working).
-    # def test_track_is_multidim(self, aircraft_data):
-    #     """
-    #     GIVEN an aircraft dataset,
-    #     WHEN passed through TimeSeries to procude a track,
-    #     THEN an iris Cube of multiple Dimension Coordinates is produced.
-    #     """
-    #     start = PartialDateTime(hour=13, minute=30)
-    #     end = PartialDateTime(hour=14, minute=30)
-    #     aircraft_track = dr.TimeSeries(aircraft_data).track(start, end)
-    #     assert len(aircraft_track.dim_coords) == 3
+    def test_track_data(self, aircraft_data):
+        """
+        GIVEN a cube of aircraft data
+        WHEN Timeseries.track() is called for valid time bounds
+        THEN an iris cube is returned
+        """
+        track_data = dr.TimeSeries(aircraft_data).track(PartialDateTime(hour=13), PartialDateTime(hour=14))
+        assert isinstance(track_data, Cube)
```

Comment:
Nice work getting this test working again (and really the functionality behind it).

I do have some edits to request though.  Although this is indeed cut down to its simplest form I find it a little unclear, so I prefer the method of setting start and end PartialDateTimes separately as it is clearer what the test is then.  Also, I would like to see a test checking the dimensionality of the output cube (as the input cube was a cube anyway, so this could be a null test if we don't also check dims).

Reply:
Good ideas, have adjusted as suggested!